app/assets/javascripts/short-form/FileUploadService.js.coffee

```diff
@@ -4,23 +4,18 @@ FileUploadService = ($http, $q, Upload, uuid, ListingDataService, ListingPrefere
   Service.preferences = {}
   Service.session_uid = -> null
 
-  Service.deletePreferenceFile = (pref_type, listing_id, opts = {}) ->
-    pref = ListingPreferenceService.getPreference(pref_type, ListingDataService.listing)
-    # might be calling deletePreferenceFile on a preference that this listing doesn't have
-    pref_id = if pref then pref.listingPreferenceID else pref_type
-    return $q.reject() unless ListingPreferenceService.getPreferenceById(pref_id, ListingDataService.listing)
-    params =
-      uploaded_file:
-        session_uid: Service.session_uid()
-        listing_id: listing_id
-        listing_preference_id: pref_id
-
-    proofDocument = Service._proofDocument(pref_type, opts)
+  Service.deleteFile = (pref_type, listing_id, opts = {}) ->
+    if pref_type
```

Comment:
wonder if we should convert pref_type to optional arg then

Reply:
let's update the snake-cased variable names in this function to be camel-cased